### PR TITLE
docs(ts): imageData optional arguments

### PR DESCRIPTION
**Pull request by sedghi**
I'm not sure how `.d.ts` types are generated in `vtk.js`, probably manually?
I found two wrong type definitions and this PR fixes them.

Affected files:
- `Sources/Common/Core/MatrixBuilder` 
- `Sources/Common/DataModel/ImageData`

### CODE REVIEW

**Comment by jourdain** [comment]
LGTM

File: Sources/Common/Core/MatrixBuilder/index.d.ts [hunk 1]
```diff
@@ -24,7 +24,7 @@ declare interface Transform {
 	 * @param {Number} angle 
 	 * @param {Number} axis
```
Comment (daker) [inline Sources/Common/Core/MatrixBuilder/index.d.ts:25]: if you can also fix the type here

File: Sources/Common/Core/MatrixBuilder/index.d.ts [hunk 2]
```diff
@@ -24,7 +24,7 @@ declare interface Transform {
 	 * @param {Number} angle 
 	 * @param {Number} axis 
 	 */
-	rotate(angle: number, axis: number): Transform
+	rotate(angle: number, axis: number[]): Transform
```
Comment (daker) [inline Sources/Common/Core/MatrixBuilder/index.d.ts:27]: you can use the types from https://github.com/Kitware/vtk-js/blob/master/Sources/types.d.ts#L20-L22

**Review by daker — CHANGES_REQUESTED** [review]

**Comment by daker** [comment]
@sedghi thanks for the fixes, yes .d.ts files are made manually, you need to change the commit scope to docs(ts): [...]

**Comment by sedghi** [comment]
@daker Thank you, I applied your comments

**Comment by daker** [comment]
@sedghi change the PR title to be the same as the commit message

**Review by daker — CHANGES_REQUESTED** [review]

File: Sources/Common/Core/MatrixBuilder/index.d.ts [hunk 3]
```diff
@@ -22,9 +22,9 @@ declare interface Transform {
 	 * Normalizes the axis of rotation then rotates the current matrix `angle`
 	 * degrees/radians around the provided axis.
 	 * @param {Number} angle 
-	 * @param {Number} axis 
+	 * @param {Number[]} axis
```
[the remark this answers is not in the document]
Comment (sedghi) [reply Sources/Common/Core/MatrixBuilder/index.d.ts]: Done, thanks

**Comment by daker** [comment]
LGTM +1

**Review by daker — APPROVED** [review]

**Comment by floryst** [comment]
LGTM

**Pull request merged**